src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GCStaticsNode.cs

```diff
@@ -73,11 +84,22 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
 
             if (factory.Target.Abi == TargetAbi.CoreRT)
             {
-                builder.EmitPointerReloc(GetGCStaticEETypeNode(factory), 1);
+                int delta = 1;
```

Comment:
1. Add `using GCStaticRegionConstants = Internal.Runtime.GCStaticRegionConstants` in the using blocks above and make this `GCStaticRegionConstants.Initialized`
2. Find out that the name of the constant is misleading.

(Can you use the named constant here and rename it to Uninitialized? Same for the 3 below - just `|` it with the other constant.)